### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by volodymyrZotov**
This PR adds additional improvements to the import item_vaults script.

- Add `--ignore-shared` option to not create items from **Shared** folders
- Print nicer logs (do not print sensitive information)
- Add additional notice to the lpass auth message saying that we do not store provided credentials
- Rename flags: `-f [--folders]` to `-d [--directory]` and `-i [--items]` to `-i [--item]`

### CODE REVIEW

File: migration/secure_note_transformer.py
Comment (edif2008) [inline migration/secure_note_transformer.py:61]: I don't think I see any place in `vault_item_import.py` a place about being unable to parse LastPass data. The only moment where we skip some LastPass data is because we don't support that conversion yet.
Is it logged somewhere still? Or can this change be better explained?

File: migration/main.py
Comment (edif2008) [inline migration/main.py:45]: Wouldn't we want to skip migrating shared folders as well if the option is set?

File: migration/vault_item_import.py
Comment (edif2008) [inline migration/vault_item_import.py:240]: Would this look better if each stat is put in a separate line?

**Review by edif2008 — COMMENTED** [review]
Code review: ✅ 

Haven't done any testing to see these improvements in action yet, so I'll come back with a comment when I've done some functional testing.

**Comment by scottisloud** [comment]
Regression? Previously these were imported but currently seeing "incompatible item" errors for a test bank account and credit card. The csv data I'm using for those items is:

```
http://sn,,,,"NoteType:Credit Card
Language:en-US
Name on Card:Scooter C. McGavin
Type:Visa
Number:4242424242424242
Security Code:123
Start Date:April,2022
Expiration Date:April,2025
Notes:For legitimate corporate spending only. NO PARTIES!",Corporate Visa,,0
http://sn,,,,"NoteType:Bank Account
Language:en-US
Bank Name:Bank of Awesome
Account Type:Chequing
Routing Number:013792
Account Number:12398123-123
SWIFT Code:93481032
IBAN Number:123123lkj1
Pin:6665
Branch Address:1739 Evergreen Terrace
Branch Phone:1-888-555-BANK
Notes:Here is a note!",Bank of Awesome Chequing,,0
```

**Comment by scottisloud** [comment]
Those items fail in #25 too with error `WARNING:root:No template generated for the Corporate Visa item. Skipping...` so I wonder if there's a regression in the template fetching side of things.

File: migration/main.py
Comment (scottisloud) [reply migration/main.py:45]: I'd agree with this. 

I took a stab at this here: https://github.com/1Password/solutions/commit/028d1df1e23fb1559dae9ce3e04cb2b3b5d02286

Did it on it's own branch because we've got two pending MRs and some bugs/regressions in both and I didn't want to muddy things up too much. If we want to cherrypick that commit over we can, but don't want to step on anyone's work in progress either.

**Comment by scottisloud** [comment]
Overall I think the improvements here are good. Most of the regressions in #25 remain here, and I haven't been able to identify any major new issues. I'm tempted to suggest we merge this into #25 and finish the fixing and polish on that branch.

**Review by scottisloud — APPROVED** [review]
Overall I think the improvements here are good. 

Since there's not a huge amount of new code here and most of the regressions in #25 remain here, and I haven't been able to identify any major _new_ issues in this MR, I'm tempted to suggest we merge this into #25 and finish the fixing and polish on that MR. 

I'll let @volodymyrZotov decide and hit the Merge button if you agree. I'll defer to your expertise on handling this flow.

File: migration/lpass.py
Comment (volodymyrZotov) [reply migration/lpass.py]: Changed

File: migration/secure_note_transformer_test.py
Comment (volodymyrZotov) [reply migration/secure_note_transformer_test.py]: Added

File: migration/vault_item_import.py
Comment (volodymyrZotov) [reply migration/vault_item_import.py]: Changed

**Comment by volodymyrZotov** [comment]
> 

Regarding the error with your csv example.
We expect it to have such columns
```
url,username,password,extra,name,grouping,fav
```
And in your example all the credit card that is in the wrong column (`name`) but suppose to be in `extra`, therefore it doesn't work as expected

**Pull request merged**
